src/nn/webgl/WebGLModel.js

```diff
@@ -278,24 +279,16 @@ export default class WebGLModel {
     });
   }
 
-  async _executeGlSubgraph(subgraph) {
-    tf.webgl.forceHalfFloat();
-    console.info('WEBGL_FORCE_F16_TEXTURES : ',tf.ENV.getBool('WEBGL_FORCE_F16_TEXTURES'));
-    console.info('floatPercision : ',tf.backend().floatPrecision());    
```

Comment:
You may need to move comment "//TF.js v1.3.1 and later use float16 as default" to here. And do you want to check the float16 precision assumption?